app/page.jsx

```diff
@@ -0,0 +1,143 @@
+            </li>
+          </ul>
+          <ul className="flex gap-2">
+            <li className={`font-semibold uppercase lg:text-white`}>login</li>
+            <li className={`font-semibold uppercase lg:text-white`}>Signup</li>
+          </ul>
+        </div>
+        <div onClick={handleNav} className="z-5 md:hidden">
+          {!nav ? (
+            <FiAlignJustify className=" z-40 relative mr-5" size={30} />
+          ) : (
+            <AiOutlineClose className="z-[20] relative  bg-white rounded-xl mr-5" size={30} />
+          )}
+        </div>
+      </div>
+      <div
+        className={
+          nav
+            ? "fixed w-[30%]  top-0 left-[20vh] h-full bg-[#5B801A] text-center lg:hidden ease-in-out duration-300 z-[10]"
+            : "fixed right-[100%]"
+        }
+      >
+        <ul className="mt-20">
+          <li className={`font-semibold mb-2 text-white`}>
+            <a href="#about">About</a>
+          </li>
+          <li className={`font-semibold mb-2 text-white`}>
+            <a href="#Home">Home</a>
+          </li>
+          <li className={`font-semibold mb-2 text-white`}>
+            <a href="#process">Process</a>
+          </li>
+          <li className={`font-semibold mb-2 text-white`}>
+            <a href="#contact">Contact</a>
+          </li>
+        </ul>
+        <ul className="mt-5">
+          <li className={`font-semibold mb-2 text-white`}>Login</li>
+          <li className={`font-semibold mb-2 text-white`}>Signup</li>
+        </ul>
+      </div>
+      <div className=" flex flex-col items-center lg:flex lg:flex-row-reverse lg:justify-between lg:ml-8 lg:mt-[-10vh]">
+        <Bin />
+        <div className="flex flex-col items-center lg:flex-row-reverse justify-around">
+          <div className="flex flex-col items-center lg:items-start lg:gap-6">
+            <h1
+              className={`font-extrabold text-3xl mb-0 text-[#5B801A] md:text-[40px] lg:text-[60px] lg:mb-2`}
+            >
+              Smart Plastic Bottle Bin:
+            </h1>
+            <h2 className={`font-extrabold text-2xl lg:text-5xl`}>
+              A Reverse Venching
+            </h2>
+            <h2 className={`font-extrabold mb-10 text-2xl lg:text-5xl`}>
+              Machine
+            </h2>
+            <Button
+              text="ABOUT US"
+              ClassName="bg-[#5B801A] w-32 h-7 text-white text-sm mb-8 gap-1 flex justify-center items-center relative lg:w-[25vh] lg:h-[5vh] lg:text-sm lg:gap-5"
+              icon={<FaArrowRight size={15} />}
+            />
+          </div>
+        </div>
+      </div>
+      <div className="flex flex-col items-center lg:flex-row h-auto w-auto gap-[7vh]">
+        <About />
+        <AboutLogo />
+      </div>
+      <div className="relative md:flex md:flex-col md:items-center">
+        <div className="p-5">
+          <Slider
+            slides={slides.map((s) => (
+              <img src={s} className="h-[200vh]" />
+            ))}
```

Comment:
lagyan mo to key tulad netong suggestion
```suggestion
            slides={slides.map((s, index) => (
              <img key={index} src={s} className="h-[200vh]" />
            ))}
```